lib/symengine/dense_matrix.rb

```diff
@@ -0,0 +1,7 @@
+module SymEngine
+  class MatrixBase
+    def inspect
+      "#<#{self.class}(#{rows}x#{cols})>"
```

Comment:
Please add specs for this `inspect`. It is quick and easy to write, and will save you from errors like `inspect`ing into something like `#<DenseMatrix(5x>` or `NoMethodErrors` or whatnot.